### PR TITLE
support lat lon (#153)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
1st step
I think I'll create another control specific to GMaps

### CODE REVIEW

**Review by dinhlongviolin1 — APPROVED** [review]
Looks simple and works like a charm

File: gui/doc/chart.csv [hunk 1]
```diff
@@ -4,9 +4,11 @@ type,indexed(str),scatter,"Chart type.
 See the Plotly <a href=""https://plotly.com/javascript/reference/"">chart type</a> documentation for details."
 mode,indexed(str),lines+markers,"Chart mode.
 See the Plotly <a href=""https://plotly.com/javascript/reference/scatter/#scatter-mode"">chart mode</a> documentation for details."
-x,indexed(str),,Column name for the <i>x</i> axis.
-y,indexed(str),,Column name for the <i>y</i> axis.
+x,indexed(str),,Column name for the <i>x</i> axis. Synonym <i>lat</i>.
```
Comment (FabienLelaquais) [inline gui/doc/chart.csv:7]: That's is quite strange.
Latitude indicates how much north or south you are (indicating a 'y' notion). And Longitude is more 'x'ish...

Rephrase: "Synonym for <i>*</i>."

**Review by FabienLelaquais — APPROVED** [review]
Looks good

File: gui/doc/chart.csv (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/chart.csv:7]: Didn't realize that :-)
I'll swap them if it makes more sense

**Pull request merged**